app/views/homepage/scanner.html.erb

```diff
@@ -0,0 +1,17 @@
+<video id="preview"></video>
+
+    <script type="text/javascript">
```

Comment:
I think it would be good to extract  `.js` to separate files so they are not mixed in with our views. We can then use: 

https://apidock.com/rails/ActionView/Helpers/AssetTagHelper/javascript_include_tag

to add these to our views